src/protocol.cpp

```diff
@@ -204,3 +204,127 @@ const std::vector<std::string> &getAllNetMessageTypes()
 {
     return allNetMessageTypesVec;
 }
+
+
+uint8_t GetShortCommandIDFromCommand(const std::string cmd)
```

Comment:
caa6c58 can reference instead of copy?
```diff
-uint8_t GetShortCommandIDFromCommand(const std::string cmd)
+uint8_t GetShortCommandIDFromCommand(const std::string& cmd)
```